src/utilities/backendValidation.ts

```diff
@@ -74,3 +75,29 @@ export default function backendValidation (fieldNames: string[], response: objec
 
   return { errorMessages, foundOnForm };
 }
+
+export default function backendValidation (fieldNames: string[], response: any): IBackendValidation {
+  if (isArray(response)) {
+    return {
+      errorMessages: [{ field: '', message: response[0] }],
+      foundOnForm: {},
+    };
+  }
+
+  if (isPlainObject(response)) {
+    const fieldErrors = getFieldErrors(response);
+    return assignErrorFieldsToFormFields(fieldNames, fieldErrors);
+  }
+
+  if (isString(response)) {
+    return {
+      errorMessages: [{ field: '', message: response }],
+      foundOnForm: {},
+    };
+  }
+
+  return {
```

Comment:
@ekatsuta Hm this is uncovered, maybe we should add a `null` test?

Reply:
Should `null` have a default message? As it stands now, wouldn't this swallow up the error?

Reply:
If we return `errorMessages: [{ field: '', message: '' }]` it will display the default.